sdk/python/feast/job_service.py

```diff
@@ -145,6 +167,54 @@ def GetJob(self, request, context):
         return GetJobResponse(job=self._job_to_proto(job))
 
 
+def run_control_loop():
+    """Runs control loop that continuously ensures that correct jobs are being run."""
+    logging.info(
+        "Feast Job Service is starting a control loop in a background thread..."
+    )
+    client = feast.Client()
```

Comment:
So how is this client different from `self.client`?

Reply:
Didn't want to use the same object from 2 different threads, and it didn't look like having 2 different client objects added a lot of overhead.